src/pages/Home/Home.js

```diff
@@ -1,146 +1,46 @@
-import { ethereum, MAIN_PAGE, menuLinks, network } from '../../utils/constants';
-
+import { useWeb3React } from '@web3-react/core';
+import Paragraph from '../../components/Paragraph';
+import MetamaskConnect from './components/MetamaskConnect';
+import StakingItem from '../../components/StakingItem';
 import CollapsedList from '../../components/CollapsedList';
-import NotSupported from '../../components/NotSupported';
 import { StakingWrapper } from '../../services/staking.wrapper';
 import { Loader } from '../../components/Loader';
 import Sidebar from '../../components/Sidebar';
-import RenderItems from '../../components/StakingItem/RenderItems';
+import RenderItems from '../../components/RenderItems';
 
 import headerLogoSvg from '../../assets/svg/header-logo.svg';
+import { MAIN_PAGE } from '../../config';
+import Menu from './components/Menu';
 
 const Home = () => {
-  const [correctNetwork, setCorrectNetwork] = useState(true);
+  const { active, chainId } = useWeb3React();
   const [pools, setPools] = useState([]);
   const location = useLocation();
   const { pathname } = location;
-  const changeNetwork = async () => {
-    if (ethereum && ethereum.isMetaMask) {
-      const provider = new providers.Web3Provider(ethereum);
-      const { chainId } = await provider.getNetwork();
-      if (chainId !== +process.env.REACT_APP_CHAIN_ID) {
-        try {
-          ethereum
-            .request({
-              method: 'wallet_addEthereumChain',
-              params: [
-                {
-                  chainId: `${utils.hexlify(+process.env.REACT_APP_CHAIN_ID)}`,
-                  chainName: `${network}`,
-                  nativeCurrency: {
-                    name: 'AMB',
-                    symbol: 'AMB',
-                    decimals: 18,
-                  },
-                  rpcUrls: [`${process.env.REACT_APP_RPC_URL}`],
-                  blockExplorerUrls: [
-                    `${process.env.REACT_APP_BLOCK_EXPLORER_URL}`,
-                  ],
-                },
-              ],
-            })
-            .then((e) => {
-              if (e) {
-                setCorrectNetwork(true);
-              }
-            });
-        } catch (e) {
-          if (e) {
-            setCorrectNetwork(false);
-          }
-        }
-      }
-    }
-  };
-  const initEthereumNetwork = async () => {
-    if (ethereum && ethereum.isMetaMask) {
-      getPulls();
-      const provider = new providers.Web3Provider(ethereum);
-      const { chainId } = await provider.getNetwork();
-      if (chainId !== +process.env.REACT_APP_CHAIN_ID) {
-        setCorrectNetwork(false);
-      }
-    } else {
-      getPulls();
-    }
-  };
+
   const getPulls = async () => {
     const stakingWrapper = new StakingWrapper();
     const poolsArr = stakingWrapper && (await stakingWrapper.getPools());
     if (poolsArr) {
       setPools(poolsArr);
     }
   };
+
+  const start = async () => {
+    await getPulls();
```

Comment:
```suggestion
    await getPools();
```

Reply:
Yes. It's me and my English, sometimes we go separately)